src/modules/FSXARemoteApi.ts

```diff
@@ -131,14 +131,23 @@ export class FSXARemoteApi implements FSXAApi {
     }
   }
 
-  private getRemoteProject(remoteProject: string) {
+  private getRemoteProjectById(remoteProjectId: string) {
```

Comment:
Yes, that should be renamed.
I guess It's named like that, because it was once meant to map a name to an id. Never actually worked that way though...